README.md

```diff
@@ -189,342 +145,299 @@ Example:
+NOTE: We do not recommend mounting the Widget on a dynamic element that's inside your main application's "controlled DOM". If you're not experienced with mixed rendering types in the Virtual DOM, you may find this challenging at first. So, to ensure success at the start, manually adding it outside of your application's controlled DOM is best. It's recommended to put the element for Widget mounting in your static HTML file as a child of the `<body>` tag.
 
-#### Instantiate the Widget (Modal)
+#### Instantiate the Widget
 
-Now, you can import the Widget into your app wherever you would like as a modal dialog (aka "lightbox"), or as an embedded component. Once the Widget is imported, you will need to instantiate it.
+You can import the Widget into your app wherever you would like as a modal dialog (aka "lightbox"), this form factor is the default. Once the Widget is imported, you will need to instantiate it.
 
 ```js
 // As modal dialog
-import Widget from '@forgerock/login-widget/modal';
+import Widget from '@forgerock/login-widget';
 
 // ...
 
+// Grab the root element added to your HTML file
+document.getElementById('widget-root');
+
+// Instantiate Widget with the `new` keyword
 new Widget({
-  target: document.getElementById('widget-root'), // Any existing element from static HTML file
-  props: {
-    config: {}, // Your JS SDK configuration; see below
-  },
+  target: widgetRootEl, // Any existing element from static HTML file
 });
 ```
 
-This mounts your Widget into the DOM. If you choose the modal version, it will be hidden at first.
+This instantiates the component and mounts the Widget into the DOM. By default, it will be the Modal version, and will therefore be be hidden at first. Component controls will be discussed shortly.
 
 Note: [See additional documentation about configuring the JS SDK](https://backstage.forgerock.com/docs/sdks/3.3/javascript/configuring/configuring-forgerock-sdk-settings-for-your-javascript-app.html).
 
-#### Starting a journey (Modal)
+#### The component lifecycle
 
-The Widget will be mounted to the DOM, but it will not display the first step of the journey. To render the first step, you'll need to import the `journey` object and call the `journey.start` method. This makes the initial request to the ForgeRock server for the initial step.
+When using the default Modal form factor, the first thing you'll want to do is open it. To do this, you need to import the `component` function. Executing this function will return a `componentEvents` object, you can name this whatever you'd like. Calling the `componentEvents.open` method will trigger the modal to animate into view.
 
 ```js
-import Widget, { journey } from '@forgerock/login-widget/modal';
+import Widget, { component } from '@forgerock/login-widget/modal';
```

Comment:
I think you mentioned it was combined now, so should this just be?:
`import Widget, { component } from '@forgerock/login-widget';`

Reply:
```suggestion
import Widget, { component } from '@forgerock/login-widget';
```